packages/volto-slate/news/5452.bugfix

```diff
@@ -0,0 +1 @@
+Fixed markup shortcut for heading and blockquote by passing 'at' to Transforms.setNodes api call for location of node @kHAPPY2004
```

Comment:
In change log entries, we need to know only what was broken and is now fixed, not _how_ it was fixed. These are narrative entries that end up in the [Volto Release Notes](https://6.docs.plone.org/volto/release-notes/index.html) where the audience is the general public, whereas the implementation details (how you fixed something) is already explained in the code and targeted to developers only.

```suggestion
In the Slate text block, the markup shortcuts for heading and blockquote work again. @kHAPPY2004
```

Reply:
I've submitted a PR that provides better guidance, with examples of good and poor change log entries at https://github.com/plone/documentation/pull/1590.